server/data/quotes.json

```diff
@@ -202,7 +202,7 @@
     {
         "author": "Haruki Murakami",
         "sentence": "Learning another language is like becoming another person."
-    },
+    },	
```

Comment:
Thanks for changing the quote! Please remove the whitespace here, and I'll merge right after.

Reply:
Thanks for reviewing, I have removed the white space on line 205 of quotes.json